### ISSUE TITLE
sport365 stream stops after a few minutes

### ISSUE THREAD

**Issue by m2840** [bugatsinho/bugatsinho.github.io#11]
Hi,
I have sport365.live addon installed in an Android box and in a Windows 8.1 PC. I have the same problem in both machines. The stream starts playing fine, but stops after a few minutes.
I don't know if the following is somehow related. If I go directly to sport365.live web site, since I have ublock origin installed, there is a notification asking to allow or block access to certain advertising sites. If I allow it, then the stream plays OK, but if I do not allow, then the stream also stops after a few minutes.
Thanks

**Comment by bugatsinho** [comment]
same issue here 
#10

**Comment by m2840** [comment]
I don't know if it is the same issue or not. The stream still stops after 6 to 7 minutes.
My guess is that your script has a problem with adshell.net.

What I was trying to say is that if, using a browser, you allow
http://tags2.adshell.net/ac.php?bannerid=58&zoneid=44&target=&withtext=0&s
to run, then the stream does not stop. If somehow it is blocked then, even in the browser, after a few minutes the stream stops.

In conclusion, for those having the same problem, there is a workaround, that is not to use Kodi addon, but to go directly to the sport365.live web site.
